templates/solutions/ai/index.html

```diff
@@ -0,0 +1,571 @@
+        <div class="u-embedded-media">
+          <iframe class="u-embedded-media__element" width="560" height="315" src="https://www.youtube.com/embed/yerW2pOczvA?si=arG1DJW8pqHzmVNU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>Why Canonical <br />for enterprise AI?</h2>
+    </div>
+    <div class="col">
+      <ul class="p-list--divided">
+        <li class="p-list__item is-ticked">Run your entire AI/ML lifecycle on a single integrated stack</li>
+        <li class="p-list__item is-ticked">Control your TCO with predictable costs and the best economics in the industry</li>
+        <li class="p-list__item is-ticked">Consume open source AI software at speed, securely</li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>One integrated AI stack</h2>
+    </div>
+    <div class="col">
+      <p>With Canonical, you get a full open source AI solution and one vendor to support it.</p>
+      <p>From GPUs that are optimised for AI use cases to a desktop environment, an integrated MLOps platform, and scalable edge deployments &mdash; do it all on a single stack.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>Fast-track compliance <br />with secure AI software</h2>
+    </div>
+    <div class="col">
+      <p>Rapidly access all the open source software you need for AI, and run it securely.</p>
+      <p>We secure and support all of the key open source software in the AI lifecycle so you don't have to. We fix Critical CVE vulnerabilities in under 24h on average, helping you streamline compliance and focus on your AI models.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>Predictable costs and optimal TCO</h2>
+    </div>
+    <div class="col">
+      <p>No software licence fees and predictable pricing per node.</p>
+      <p>Enjoy the freedom of open source, with the best support and economics in the industry.</p>
+      <hr />
+      <p>
+        <a href="/contact-us" aria-controls="ai-modal">Get in touch to kickstart your journey&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+  </div>
+</section>
+
+<hr class="is-fixed-width" />
+<section class="p-strip is-deep">
+  <div class="u-fixed-width">
+    <h2>
+      <a href="https://ubuntu.com/engage/open-source-big-data-ai">Download our guide: Build a smarter enterprise with big data and AI</a>
+    </h2>
+  </div>
+</section>
+<hr class="is-fixed-width" />
```

Comment:
This likely should be a `p-rule` inside the next section.

Now, as `hr` it has additional margin which likely shouldn't be there between line and following text.